ansible/roles/couchdb/tasks/main.yml

```diff
@@ -33,9 +33,18 @@
     - tar xvf ~/{{ couchdb_distro_filename }}
     - cd apache-couchdb-{{ couchdb_version }} && ./configure
     - make -C ~/apache-couchdb-{{ couchdb_version }}
-    - sudo make install -C ~/apache-couchdb-{{ couchdb_version }}
-    - rm -rf ~/apache-couchdb-{{ couchdb_version }}*
   when: not couchdb_path.stat.exists
+  register: couchdb_compile
+
+- name: Install CouchDB
+  shell: "{{ item }}"
+  args:
+    chdir: ~/
+  sudo: yes
```

Comment:
Compiling is done without `sudo` so as to avoid permission problems see [commit message](https://github.com/dimagi/commcarehq-ansible/commit/0c2951a3f7ddfee0c854a907dcd18ef39b018a3f). But installation requires sudo as shown in the command here. If sudo is used via raw commands, ansible hangs at the password prompts. sudo commands should be done via `sudo: yes`. So this PR seperates out the part that needs sudo into a separate sudo task